app/src/main/java/ru/suslovalex/androidacademyapp/FragmentMoviesDetails.kt

```diff
@@ -4,10 +4,20 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.RatingBar
 import android.widget.TextView
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.RecyclerView
+import ru.suslovalex.androidacademyapp.adapters.AdapterActors
+import ru.suslovalex.androidacademyapp.model.Movie
+import ru.suslovalex.androidacademyapp.repository.getListActors
+import ru.suslovalex.androidacademyapp.repository.getListMovies
 
-class FragmentMoviesDetails: Fragment() {
+class FragmentMoviesDetails : Fragment() {
+
+    private lateinit var currentMovie: Movie
+    private lateinit var adapterActors: AdapterActors
```

Comment:
@AleksandrAlex Looks like no need. You can easy do like this `recyclerView.adapter = AdapterActors(actors)` and pass actor to the constructor